src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/util/DataLoadUtil.java

```diff
@@ -49,4 +50,28 @@ public static void registerFileStatusBean(ApplicationContext applicationContext,
             beanFactory.registerSingleton(fileName, fileStatus);
         }
     }
+
+    public static boolean isStringArraysEqual(String[] expected, String[] actual) {
+        BiPredicate<String[], String[]> nullCheck = (exp,act) -> (expected != null && actual != null);
+        BiPredicate<String[], String[]> sizeCheck = (exp,act) -> (act.length == exp.length);
+        BiPredicate<String[], String[]> arrayEqualsIgnoreCaseCheck = DataLoadUtil::isArraysEqual;
+
+        boolean isChecksPass = nullCheck.and(sizeCheck).test(expected, actual);
+        if (!isChecksPass) {
```

Comment:
return nullCheck.and(sizeCheck).and(arrayEqualsIgnoreCaseCheck).test(expected, actual); I think you can do like this